build/maintainer-flags.txt

```diff
@@ -1,11 +1,9 @@
 -Wall
--Wdeclaration-after-statement
 -Wempty-body
 -Wformat
 -Wformat-nonliteral
 -Wformat-security
 -Winit-self
--Winline
```

Comment:
This warnings is issued when GCC decides not to inline an `extern inline` function. It's inherently meaningless in modern code where the inliner (usually) knows better. This can be reverted for now because only `mlib` contained `extern inline` functions.